superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -38,6 +38,22 @@
   }
 }
 
+.dashboard .dashboard-header {
+  #save-dash-split-button {
+    border-radius: 0;
+    margin-left: -5px;
+    padding: 6px;
+    height: 30px;
+    width: 30px;
+
+    .caret {
+      position: absolute;
```

Comment:
can you avoid absolute positioning? like use flexbox?

Reply:
.caret inherit many styles, if not use `position: absolute;` it's just too many things to overwrite.